visualization/app/codeCharta/ui/attributeSideBar/util/metricDescriptors.ts

```diff
@@ -1,17 +1,23 @@
 import { AttributeDescriptors } from "../../../codeCharta.model"
 import { metricTitles } from "../../../util/metric/metricTitles"
 
-export type LegendMetric = {
+export type MetricDescriptors = {
 	key: string
 	title: string
 	description: string
+	hintLowValue: string
+	hintHighValue: string
+	link: string
 }
 
-export function getLegendMetric(metricKey: string, attributeDescriptors: AttributeDescriptors): LegendMetric {
+export function getMetricDescriptors(metricKey: string, attributeDescriptors: AttributeDescriptors): MetricDescriptors {
 	return {
 		key: metricKey,
 		title: getMetricTitle(metricKey, attributeDescriptors),
-		description: attributeDescriptors?.[metricKey]?.description
+		description: attributeDescriptors?.[metricKey]?.description,
+		hintLowValue: attributeDescriptors?.[metricKey]?.hintLowValue,
+		hintHighValue: attributeDescriptors?.[metricKey]?.hintHighValue,
+		link: attributeDescriptors?.[metricKey]?.link
```

Comment:
Are all those values supposed to be nullable?
Because we should try to enforce type-safety whenever possible

Reply:
Since neither the AttributeDescriptors field nor the individual descriptions must be present, so the representation over nullable attributes makes sense for me altogether. 
However, I will think again about the description structure inside the model when im working on the display of them in the sidebar and might refactor some parts.

Reply:
But, I think I could remove the first safe chain operator?

Reply:
If neither field are required then a safe chain is a legitimate solution
It was more of a question as I considered the attributeDescriptors field a required property, but if this models the data then its a good solution! :)